acme_diags/parameter/enso_diags_parameter.py

```diff
@@ -11,6 +11,18 @@ def __init__(self):
         self.ref_timeseries_input = True
         self.test_timeseries_input = True
 
+        if hasattr(self, 'start_yr'):
```

Comment:
If `test_start_yr` is set, use that value. Else, use `start_yr`. Similar logic for the other three parameters.

Reply:
The below lines in `check_value` function need to be updated accordingly.
```     
   if not (hasattr(self, 'start_yr') and hasattr(self, 'end_yr')):
            msg = "You need to define both the 'start_yr' and 'end_yr' parameter."
            raise RuntimeError(msg)
```

Reply:
What exactly needs to be checked? If one is defined, the other still needs to be. 

If `start_yr` isn't defined but `test_start_yr` and `ref_start_yr` are equal, then I suppose we could also set `start_yr` to be that value.